README.md

```diff
@@ -71,10 +71,10 @@ The template can be as the following example:
 After the initial setup every time a change is made, the developer should run the following command in the project root path:
 
 ``` bash
-$ codelog new
+$ codelog new NAME
 ```
 
-This will generate a change file on `changelogs/unreleased/` from the `template.yml` named with a timestamp value followed by `_change.yml`.
+This will generate a change file on `changelogs/unreleased/` from the `template.yml` named with a timestamp value followed by the given `NAME` or the default value.
```

Comment:
Was it a intentional change to remove the default suffix value from the documentation? I would prefer to know what name would be given by default. I suggest something like _"the given `NAME` or the default value (`_change.yml`)."_, what do you think?

Reply:
I changed the message to 
```
the given `NAME`, converted to snake case, or the default value(`change`), with the suffix `_yml`
```
is it ok for you?

Reply:
I think the suffix `_yml` not added; its the file's extension.

I would change 
> or the default value(`change`)

to
> or the default name(`change`)